content/docs/operating/configuration.md

```diff
@@ -399,7 +399,7 @@ Prometheus relabeling to control which instances will actually be scraped. Also
 by default all apps will show up as a single job in Prometheus (the one specified
 in the configuration file), which can also be changed using relabeling.
 
-### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+### `<serverset_sd_config>`
```

Comment:
I wonder if we should remove the `<` `>` from the headers/links. Github (and I suspect others) will remove them from links as they look like a HTML tag.